Header header
uint32 frame_count
uint16 score_ind
float32 score
